docs/docs/ab_testing.md

```diff
@@ -0,0 +1,106 @@
+Pathfora supports A/B testing on a global level as well as on a targeted audience. You can split a pool of users to show some proportion of them a group of modules "A" and the other portion a group "B".
```

Comment:
probably worth mentioning that right now we only support 50/50, 80/20 and 100 splits